CS342MemberViews/Sources/ShriyaReddy/ShriyaReddy.swift

```diff
@@ -0,0 +1,59 @@
+//
+// File.swift
+// Created by Shriya Reddy on 1/19/23.
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+let backgroundGradient = LinearGradient(
+    colors: [
+        Color.purple,
+        Color.white
+    ],
+    startPoint: .top,
+    endPoint: .center
+)
```

Comment:
Like noted in the `HobbyButton`: Please add properties (including this computed properties) into the scope of a type that they are used to, e.g., the `HobbyButton` view. You can simply move it into the struct by adding it above or below the `body` computed propert.

You have correctly added the description property to the `IntroButton` in the right way.